torch/distributed/distributed_c10d.py

```diff
@@ -183,13 +182,7 @@ def __new__(cls, name: str):
             raise ValueError("Backend name must be a string, but got: {}".format(name))
         value = getattr(Backend, name.upper(), Backend.UNDEFINED)
 
-        if value == Backend.TCP:
-            raise ValueError(
-                "TCP backend has been deprecated. Please use "
-                "Gloo or MPI backend for collective operations "
-                "on CPU tensors."
-            )
-        elif value != Backend.GLOO and value != Backend.NCCL and value != Backend.UCC and value != Backend.MPI:
+        if value != Backend.GLOO and value != Backend.NCCL and value != Backend.UCC and value != Backend.MPI:
```

Comment:
nit: (Before this PR) Since tuple equality checks should use `==` IIUC, consider the following for brevity:
```suggestion
        if value not in (Backend.GLOO, Backend.NCCL, Backend.UCC, Backend.MPI):
```

Reply:
added. Probably goign to remove this line later since its for 3rd party ext. and will b updated